Docs/source/usage/parameters.rst

```diff
@@ -158,8 +158,19 @@ Setting up the field mesh
 
     Note: in development; currently allowed value: ``2 2 2``.
 
-* ``geometry.coord_sys`` (`integer`) optional (default `0`)
-    Coordinate system used by the simulation. 0 for Cartesian, 1 for cylindrical.
+* ``geometry.dims`` (`string`)
+    The geometry of the simulation.
```

Comment:
```suggestion
    The dimensions of the simulation geometry.
```

Reply:
The geometry may be confusing with the physical coordinates?

Reply:
Sounds good to me :) Thx!!